cmd/lakectl/cmd/annotate.go

```diff
@@ -0,0 +1,53 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/treeverse/lakefs/pkg/api"
+)
+
+const annotateTemplate = `{{ range $val := .Commits}}
+{{ $.Object|ljust 10}} {{ $val.Committer }} {{ $val.Id | printf "%.16s"|ljust 20 }} {{ $val.Message |ljust 30 }} {{ $val.CreationDate|date }}{{ end -}}
```

Comment:
Commit Message is much more likely than other fields to be very long and to contain line breaks, so I suggest:

1. moving it to be the last item per line - it will also help anyone looking to parse the output of the command
1. redacting it after say 100 chars (and adding a trailing "...")
1. in case of a line break (\r\n or \n), redact just before it, with a trailing "...".

Reply:
1,2. Good point. Done.
3. I do agree with the redact by line breaks, though I think adding a trailing "...." in this case is not needed since the text itself is formatted in the proper form while blame should only add the most basic info of the commit.